src/test/kotlin/no/nav/syfo/testutil/TestData.kt

```diff
@@ -70,8 +70,8 @@ fun getSykmeldingDto(skjermet: Boolean = false): SykmeldingDTO {
             legekontorOrgnummer = null,
             andreTiltak = null,
```

Comment:
I metoden fun getSykmeldingerDBmodel
så kan man returene objekete med ein gang istende for å lage ein variabel ut av det, for å så returnere det